Oracle customization - non upstream patch
Modify hiera dependency to use json instead of json_pure

--- hiera-1.3.4/hiera.gemspec.orig	2014-07-17 14:19:57.751752120 -0600
+++ hiera-1.3.4/hiera.gemspec	2014-07-17 14:20:40.070346040 -0600
@@ -14,7 +14,7 @@
 date: 2014-06-06 00:00:00 Z
 dependencies: 
 - !ruby/object:Gem::Dependency 
-  name: json_pure
+  name: json
   prerelease: false
   requirement: !ruby/object:Gem::Requirement 
     requirements: 
